### PR TITLE
Add new formatting setting `xml.format.splitAttributesIndentSize`

### PR DESCRIPTION
**Pull request by datho7561**
Add a new setting which controls the level of indentation of attributes
with respect to their parent element when `xml.format.splitAttributes`
is set to `true`.

Based off of a comment by Cong Wang (@I322871) on the vscode-xml Gitter

Signed-off-by: David Thompson <davthomp@redhat.com>

### CODE REVIEW

**Comment by datho7561** [comment]
Please see the server side changes: eclipse/lemminx#952

File: docs/Formatting.md [hunk 1]
```diff
@@ -148,7 +148,8 @@ If this value is set to 0, then all blank lines will be removed during formattin
 ### xml.format.splitAttributes
 
   Set to `true` to split node attributes onto multiple lines during formatting. Defaults to `false`.
-  Overrides the behaviour of [xml.format.preserveAttributeLineBreaks](#xmlformatpreserveattributelinebreaks)
+  Overrides the behaviour of [xml.format.preserveAttributeLineBreaks](#xmlformatpreserveattributelinebreaks).
+  Please see [xml.format.numberOfIndentsForSplitAttributes](#xmlformatnumberOfIndentsForSplitAttributes) for information on configuring the indentation level of the attributes.
```
Comment (fbricon) [inline docs/Formatting.md:152]: splitAttributesIndentation is a tiny bit shorter

Comment (angelozerr) [reply docs/Formatting.md:152]: HTML LS provides a similar settings with `html.format.wrapAttributesIndentSize`, to be consistent with HTML, I suggest `xml.format.spliAttributesIndentSize`.

File: package.json [hunk 2]
```diff
@@ -257,6 +257,11 @@
           "markdownDescription": "Split multiple attributes each onto a new line. Default is `false`. See [here](command:xml.open.docs?%5B%7B%22page%22%3A%22Formatting%22%2C%22section%22%3A%22xmlformatsplitattributes%22%7D%5D) for more information.",
           "scope": "window"
         },
+        "xml.format.numberOfIndentsForSplitAttributes": {
+          "type": "number",
+          "default": 2,
+          "markdownDescription": "How many levels to indent the attributes by when `xml.format.splitAttributes` is `true`. See [here](command:xml.open.docs?%5B%7B%22page%22%3A%22Formatting%22%2C%22section%22%3A%22xmlformatnumberofindentsforsplitattributes%22%7D%5D) for more information"
```
Comment (angelozerr) [inline package.json:263]: `when `#xml.format.splitAttributes#`` to benefit with hyperlink in UI settings.

**Comment by datho7561** [comment]
- [ ] Rename commit

File: package.json [hunk 3]
```diff
@@ -257,6 +257,11 @@
           "markdownDescription": "Split multiple attributes each onto a new line. Default is `false`. See [here](command:xml.open.docs?%5B%7B%22page%22%3A%22Formatting%22%2C%22section%22%3A%22xmlformatsplitattributes%22%7D%5D) for more information.",
           "scope": "window"
         },
+        "xml.format.splitAttributesIndentSize": {
+          "type": "number",
+          "default": 2,
+          "markdownDescription": "How many levels to indent the attributes by when `#xml.format.splitAttributes#` is `true`. See [here](command:xml.open.docs?%5B%7B%22page%22%3A%22Formatting%22%2C%22section%22%3A%22xmlformatsplitattributesindentsize%22%7D%5D) for more information"
```
Comment (angelozerr) [inline package.json:263]: Before . See [here], Add `Default value is 2.`

**Comment by angelozerr** [comment]
@datho7561 the new `xml.format.splitAttributesIndentSize` must be added in the README too

File: package-lock.json [hunk 4]
```diff
@@ -2243,12 +2243,12 @@
           "dev": true,
           "optional": true,
           "requires": {
-            "minimist": "^1.2.5"
+            "minimist": "0.0.8"
```
Comment (datho7561) [inline package-lock.json:2246]: oh no

File: README.md [hunk 5]
```diff
@@ -70,15 +70,16 @@ The following settings are supported:
...
-* [`xml.format.spaceBeforeEmptyCloseTag`](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Formatting.md#xmlformatspacebeforeemptyclosetag): Insert space before end of self closing tag.  
+* [`xml.format.spaceBeforeEmptyCloseTag`](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Formatting.md#xmlformatspacebeforeemptyclosetag): Insert space before end of self closing tag.
   Example: ```<tag/> -> <tag />```. Default is `true`.
 * [`xml.format.splitAttributes`](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Formatting.md#xmlformatsplitattributes): Split multiple attributes each onto a new line. Default is `false`.
+* [`xml.format.splitAttributesIndentSize`](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Formatting.md#xmlformatsplitattributesindentsize): How many levels to indent the attributes by when `xml.format.splitAttributes` is `true`. Default value is 2.
```
Comment (angelozerr) [inline README.md:80]: Default value is `2`.

**Review by angelozerr — APPROVED** [review]

**Comment by angelozerr** [comment]
Thanks @datho7561 !

**Pull request merged**